src/ui/qgsprojectpropertiesbase.ui

```diff
@@ -2370,7 +2370,7 @@
                           <item>
                            <widget class="QLabel" name="mWMSImageQualityLabel">
                             <property name="text">
-                             <string>Quality for JPEG images ( 10 : smaller image - 100 : best quality )</string>
+                             <string>Quality for JPEG ( 10 : smaller image - 100 : best quality ) and WebP images</string>
```

Comment:
Also 10-100?

Reply:
From 0 to 100, I redid the string to include this. Thanks!

Reply:
Thanks

1. there's a problem in formatting now (closing paranthesis for the first one), can you fix that?
2. the spacing around parantheses is different
3. it doesn't say for WebP which one is smaller in size/better quality, ideally we have the same level of information for all parts. If they are similar enough for JPEG and WebP, we can also just mention it once.

Reply:
Done, sorry for the noise. Thank you.